airesources/Scala/hlt/Constants.scala

```diff
@@ -34,15 +34,15 @@ object Constants {
   /** Number of turns it takes to dock a ship */
   val DOCK_TURNS = 5
 
-  /** Number of production units per turn contributed by each docked ship */
+  /** Number of turns it takes to create a ship per docked ship */
```

Comment:
Might be a merge issue but, the previous phrase was correct. This is the production units produced per docked ship per turn, not the turns needed to produce a ship per docked ship.